compiler/gradient_with_order.cc

```diff
@@ -202,10 +257,9 @@ void AddGradientNodesForTrainingWithOrders(Graph* graph, const std::vector<Order
                     }
                 }
 
-                ScheduleAddedScope schedule_scope(graph, schedule_node);
-                if (!AddGradientForNode(graph, graph, node, nullptr)) {
+                ScheduleAddedScope schedule_scope(bwd_graph, schedule_node);
+                if (!AddGradientForNode(bwd_graph, bwd_graph, node, nullptr)) {  // NOTE: first argument may be fwd_graph?
```

Comment:
Yeah, this the first argument should be `fwd_graph`.

Reply:
It seems that, if we change the first argument to `fwd_graph`, `runtests.py` yields more filed tests than current one.

```
# When first argument = bwd_graph:

$ python ./scripts/runtests.py --computation_order=dummy
...
65/976 tests failed! (see out/failed_tests.log)

# When first argument = fwd_graph:

$ python ./scripts/runtests.py --computation_order=dummy
...
75/976 tests failed! (see out/failed_tests.log)
```

Honestly, I do not totally figure out what is correct. I may seek what is really correct in future.